proto/greptime/v1/meta/ddl.proto

```diff
@@ -61,4 +69,7 @@ message SubmitDdlTaskResponse {
     bytes key = 2;
 
     Status status = 3;
+
```

Comment:
Is there a failed message when the status is Failed?

Reply:
I removed the `Status` struct. For the sync version, it only has two statuses, Done or failed. We can use the error msg in the `ResponseHeader.`